src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -94,6 +94,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val sourceReader = StringSetting("-Xsource-reader", "classname", "Specify a custom method for reading source files.", "")
   val XnoValueClasses = BooleanSetting("-Xno-value-classes", "Do not use value classes. Helps debugging.")
   val XreplLineWidth = IntSetting("-Xrepl-line-width", "Maximial number of columns per line for REPL output", 390)
+  val XreplNoColor = BooleanSetting("-Xrepl-no-color", "Removes syntax highlighting in REPL")
```

Comment:
The name doesn't really matter, but I think you should phrase the option positively (`-Xrepl-color`, or just `-color` since this isn't an "advanced" option) and you really want three possible values for color: `never`,`auto` and `always`, `auto` means that we try to guess if we're in a terminal (as opposed to redirected to a file) and only if that's the case do we enable colors, this is how color is specified in many cli tools (gcc `-fdiagnostics-color`: https://gcc.gnu.org/onlinedocs/gcc-5.3.0/gcc/Language-Independent-Options.html, git, ...). But this isn't needed for this PR of course.